### PR TITLE
chore: exclude non-`values` translation files in Android Studio Index/Search

### PR DESCRIPTION
**Pull request by david-allison**
## Purpose / Description
* Hiding files came up as a suggestion to help user onboarding
* I personally would find AnkiDroid easier to navigate if searching for a string constant only showed the items in `/values`
* This hopefully speeds up the IDE as it removes a large number of files from the index

## Approach

`Exclude` all non-`values` folders: `values-af` etc...

* This removes them from the search/index
* This marks them in the IDE in a different color, making it less likely a new user will modify them
  * Files also now contain IDE errors

## How Has This Been Tested?
Tested on an API 33 emulator and languages are still switchable

**Translations are implied by the IDE as non-editable**
excluded-values-ankidroid

**IDE Search is much better**
excluded-values-search

**Android Studio RAM Usage: 2.38 -> 2.19 GB**

## Learning (optional, can help others)
We go against JetBrains advice as we want these files in Gradle:

> Items you may want to exclude from sharing:
>
> .iml files and .idea/modules.xml file for the Gradle or Maven based
> projects since these files will be generated on import

https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

----

I did test `Editor > File Types > Ignored files and folders` This removes the files from the IDE completely
but I could not find a way to share this with other team members

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: .idea/modules/AnkiDroid/Anki-Android.AnkiDroid.main.iml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$/../../../AnkiDroid/src/main">
+      <excludeFolder url="file://$MODULE_DIR$/../../../AnkiDroid/src/main/res/values-af" />
```
Comment (david-allison) [inline .idea/modules/AnkiDroid/Anki-Android.AnkiDroid.main.iml:5]: Note: I copied the folder names, and excluded non-language ones (`v29`: I double-checked, but may have included a non-language code here)

File: .idea/modules/AnkiDroid/Anki-Android.AnkiDroid.main.iml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      <excludeFolder url="file://$MODULE_DIR$/../../../AnkiDroid/src/main/res/values-am" />
+      <excludeFolder url="file://$MODULE_DIR$/../../../AnkiDroid/src/main/res/values-ar" />
+      <excludeFolder url="file://$MODULE_DIR$/../../../AnkiDroid/src/main/res/values-az" />
+      <excludeFolder url="file://$MODULE_DIR$/../../../AnkiDroid/src/main/res/values-be" />
+      <excludeFolder url="file://$MODULE_DIR$/../../../AnkiDroid/src/main/res/values-bg" />
```
Comment (david-allison) [inline .idea/modules/AnkiDroid/Anki-Android.AnkiDroid.main.iml:10]: Note: I did have my IDE reorder this file. I can't reproduce this and I hope it's stable

Testing this should be a priority

**Review by BrayanDSO — APPROVED** [review]
LGTM. Great addition

**Comment by david-allison** [comment]
Just to note: would be great to mark these files as auto generated. 

The IntelliJ syntax doesn't work in Android Studio, but it may be possible in gradle.

Optional extension, as this PR feels like strong incremental DevEx progress

**Review by mikehardy — APPROVED** [review]
we do a couple other usually-not-advised things for specific / well-reasoned purposes, and it hasn't really bitten us before

My personal policy is "if it seems like it might work, try it? you can always revert" for devex type iffy items. So I'm more than happy to give it a go

**Pull request merged**